splipy/basis_test.py

```diff
@@ -228,5 +228,33 @@ def test_make_periodic(self):
         self.assertAlmostEqual(c.knots[9], 7)
         self.assertAlmostEqual(c.knots[10], 8)
 
+    def test_snap(self):
+        b = BSplineBasis(3, [0,0,0, .123, np.pi, 3.5, 4,4,4])
+        t = [0,.123+1e-12, 3, 3.1, 3.14, 3.1415, 3.1415926, 3.500000000000002, 4+1e-12, 4+1e-2]
+        b.snap(t)
+        self.assertEqual(   t[0], b[0]) # 0
+        self.assertEqual(   t[1], b[3]) # 0.123
+        self.assertNotEqual(t[2], b[4]) # 3
+        self.assertNotEqual(t[3], b[4]) # 3.1
+        self.assertNotEqual(t[4], b[4]) # 3.14
+        self.assertNotEqual(t[5], b[4]) # 3.1415
+        self.assertNotEqual(t[6], b[4]) # 3.1415926
+        self.assertEqual(   t[7], b[5]) # 3.5
+        self.assertEqual(   t[8], b[6]) # 4.0
+        self.assertNotEqual(t[9], b[6]) # 4.01
+
+    def test_bug44(self):
+        # https://github.com/sintefmath/Splipy/issues/44
+        b = BSplineBasis(4, [ 0.        , 0.        , 0.        , 0.        , 0.01092385, 0.02200375,\
```

Comment:
Python has implicit line-continuation inside bracket pairs, `()`, `[]` and `{}`. So no need for a backslash here.